server/src/models/User.js

```diff
@@ -43,6 +43,33 @@ class User extends uniqueFunc(Model) {
 
     return serializedJson;
   }
+
+  static get relationMappings() {
+    const { Movie, Review } = require("./index.js");
+
+    return {
+      movies: {
```

Comment:
only need this association if we are using it to see "all the Movies that a User has left a Review for"